SPECS/debugedit/debugedit.spec

```diff
@@ -3,14 +3,16 @@ Name:           debugedit
 Version:        5.0
 Release:        3%{?dist}
 License:        GPLv3+
-URL:            https://sourceware.org/debugedit/
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
+URL:            https://sourceware.org/debugedit/
 Source0:        https://sourceware.org/ftp/%{name}/%{version}/%{name}-%{version}.tar.xz
 Patch0:         BUG-28161.patch
 BuildRequires:  automake
 BuildRequires:  binutils
+%if %{with_check}
```

Comment:
Tried this out, but the help2man missing issue still exists in the toolchain's 2nd phase. Seems like `with_check` may be set when the toolchain phase 2 runs?

Reply:
Yep. I saw some errors locally and in the buddy builds as well. It looks like the patch makes some changes that require help2man to be available.